src/e2e-test/features/recordsplitter/GCSToRecordSplitter.feature

```diff
@@ -0,0 +1,58 @@
+@RecordSplitter
+Feature: RecordSplitter transform - Verify GCS source data transfer using RecordSplitter transform
+
+  @GCS_CSV_DELIMITED_TEST @BQ_SINK_TEST
+  Scenario: To verify data is getting transferred from GCS source to BigQuery sink with record splitter
```

Comment:
we can remove this test case as it's already covered by previous test cases (no need to test different combinations of sources and sinks as focus is on testing the record splitter transform here).

Reply:
Removed.